llama_index/callbacks/wandb_callback.py

```diff
@@ -250,8 +250,9 @@ def log_trace_tree(self) -> None:
                 if self._wandb.run:
                     self._wandb.run.log({"trace": root_trace})
                 self._wandb.termlog("Logged trace tree to W&B.")
-        except:  # noqa
-            # Silently ignore errors to not break user code
+        except Exception as e:
+            print(f"Failed to log trace tree to W&B: {e}")
```

Comment:
is this an intentional change?

Reply:
Yea, silently failing is very annoying and non-obvious to the user. Took me ages to find out what was wrong lol

It will still not raise an error though